packages/golden_toolkit/lib/src/golden_builder/golden_builder_table.dart

```diff
@@ -18,6 +18,7 @@ class GoldenBuilderTable extends GoldenBuilder {
     this.columnSpacing = 16.0,
     this.columnWidths = const {},
     super.wrap,
+    super.titleTextStyle,
```

Comment:
add the parameter to `Single` and `Grid` builder, to be consistent
`Grid` may end up being deprecated in the future, in favor of `Table`, for yeah...
having a title in `Single` is kinda redundant, since it only contains 1 element, but, either way, allow the constructor to receive it